Testsuite: gov.nasa.jpf.test.vm.basic.InitializeInterfaceClassObjectRefTest
Tests run: 1, Failures: 0, Errors: 0, Time elapsed: 0.536 sec
------------- Standard Output ---------------
  running jpf with args: +log.finest+=,gov.nasa.jpf.vm.ClassInfo
JavaPathfinder v7.0 (rev 1124) - (C) RIACS/NASA Ames Research Center


====================================================== system under test
gov.nasa.jpf.test.vm.basic.InitializeInterfaceClassObjectRefTest.runTestMethod()

====================================================== search started: 10/23/13 9:33 PM

====================================================== error 1
gov.nasa.jpf.vm.NoUncaughtExceptionsProperty
java.lang.RuntimeException: This test throws an expected exception.
	at gov.nasa.jpf.test.vm.basic.InitializeInterfaceClassObjectRefTest.test(InitializeInterfaceClassObjectRefTest.java:41)
	at java.lang.reflect.Method.invoke(gov.nasa.jpf.vm.JPF_java_lang_reflect_Method)
	at gov.nasa.jpf.util.test.TestJPF.runTestMethod(TestJPF.java:603)


====================================================== snapshot #1
thread java.lang.Thread:{id:0,name:main,status:RUNNING,priority:5,lockCount:0,suspendCount:0}
  call stack:
	at gov.nasa.jpf.util.test.TestJPF.runTestMethod(TestJPF.java:605)


====================================================== results
error #1: gov.nasa.jpf.vm.NoUncaughtExceptionsProperty "java.lang.RuntimeException: This test throws an ex..."

====================================================== search finished: 10/23/13 9:33 PM
------------- ---------------- ---------------

Testcase: test took 0.521 sec
